README.md

```diff
@@ -3,6 +3,11 @@
 # Portfolio
 
 > A great web site to show off my works
+Visit my website [here](https://kosher9.github.io/)
+
+Take a look at the [mobile](portfolio/img/scshot_1.png) & [desktop](portfolio/img/scshot_2.png) version
```

Comment:
- [ ] Kindly remove or add the right links current the links are broken

- [ ] You can also add a section for your deployed link so it if easily found by people check suggestion below, this will make your project more professional

```suggestion
## live demo
[live demo](https://kosher9.github.io/)
```